assets/src/components/parts/janus-capi-iframe/schema.ts

```diff
@@ -19,9 +19,33 @@ export const schema: JSONSchema7Object = {
 
 export const getCapabilities = () => ({
   configure: true,
-  save: true,
 });
 
+export const adaptivitySchema = ({
```

Comment:
it is supposed to be an async function, though I suppose it doesn't matter until you use async inside it